### DISCUSSION TITLE
Change code block theme according to the Light/Dark mode

### DISCUSSION THREAD

**Discussion by iqfareez** [timlrx/tailwind-nextjs-starter-blog#509] [Q&A]
Hi, I'm aware of we can customize the look of code block via `prism.css`. But, I'm wondering if I can change the code block style based on the light/dark mode of the website.

Thank you.

**Comment by eduardo-escoto** [comment]
This is definitely a pain point for me as well! Would be awesome if we could do this. 👍🏼

**Comment by dlarroder** [comment]
You can use tailwind to style your code blocks in `prism.css`

For example:

```css
.token.constant,
.token.function,
.token.builtin,
.token.char {
  @apply text-gray-400 dark:text-white;
}
```

This should result to: 

https://user-images.githubusercontent.com/52998821/179805704-d43b86dc-b0e1-405c-8fa0-c6d0704fd0d4.mov


Hope this helps!
